src/index.css

```diff
@@ -25,7 +25,3 @@ code {
     'Courier New',
     monospace;
 }
-
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
```

Comment:
Please, recover this `tailwindcss` set-up. it is necessary for those who used it in this project.